src/test/scala/uk/gov/hmrc/test/ui/pages/CommonClass.scala

```diff
@@ -75,38 +83,60 @@ object CommonClass extends BasePage {
     this
   }
 
+  def cancelErrorMessageValidation(EORI: String): this.type = {
```

Comment:
this kind of validation should be covered under unit test level, so can be removed from AT.